packages/ubuntu_wizard/pubspec.yaml

```diff
@@ -35,6 +38,7 @@ dependencies:
       url: https://github.com/canonical/ubuntu-flutter-plugins.git
       path: packages/ubuntu_widgets
   url_launcher: ^6.1.0
+  url_launcher_platform_interface: ^2.0.5
```

Comment:
only used for testing url launcher -> move to `dev_dependencies`

Reply:
✅

Reply:
✅